### PR TITLE
api/test: fix test runner

### PR DESCRIPTION
**Pull request by chicoxyzzy**
All the tests should run on the same tick, otherwise they don't run at all

Better review with hidden spaces: https://github.com/socketsupply/socket/pull/143/files?diff=split&w=1

### CODE REVIEW

File: test/src/index.js [hunk 1]
```diff
@@ -13,9 +12,5 @@ import './dns.js'
 import './backend.js'
 import './crypto.js'
 import './util.js'
-
-// TODO(@jwerle): FIXME
-if (os.platform() !== 'win32') {
-  await import('./runtime.js')
-  await import('./fs.js')
-}
```
Comment (chicoxyzzy) [inline test/src/index.js:21]: these tests should run in the same tick

File: api/test/index.js [hunk 2]
```diff
@@ -188,7 +188,6 @@ export class Test {
     try {
       fn()
     } catch (err) {
-      // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```
Comment (chicoxyzzy) [inline api/test/index.js:191]: otherwise standard yells on me

**Comment by chicoxyzzy** [comment]
Windows Ci never fail on tests because we can't get the exit code 1 from the test app. I will remove win check from the PR.

File: test/src/index.js [hunk 3]
```diff
@@ -13,9 +12,5 @@ import './dns.js'
...
+import './runtime.js'
```
Comment (jwerle) [inline test/src/index.js:15]: we had this check here for windows, why is it removed? imports are not synchronous so with

Comment (chicoxyzzy) [reply test/src/index.js:15]: I moved these checks to the modules. Tests may end with `exit 0` early without this like [here](https://github.com/socketsupply/socket/actions/runs/4099615181/jobs/7069821754). There are failing tests, but no exit 1 as runner exits on some of previous ticks

Comment (chicoxyzzy) [reply test/src/index.js:15]: so this gives a false positive

**Pull request merged**